pkg/commands/compute/pack.go

```diff
@@ -39,6 +39,8 @@ func (c *PackCommand) Exec(_ io.Reader, out io.Writer) (err error) {
 	progress := text.NewProgress(out, c.Globals.Verbose())
 
 	defer func(errLog fsterr.LogInterface) {
+		os.RemoveAll("pkg/package")
```

Comment:
ℹ️ In the `compute build` command we create a 'temp' directory to store files that are placed into the final `package.tar.gz`, and then clean up the temp directory. So we are now doing the same thing here, and are just making sure we clean-up after ourselves.